### ISSUE TITLE
Consider integrating Fotorama.io slideshow/viewer - touch-device support

### ISSUE THREAD

**Issue by oparoz** [nextcloud/gallery#23]
_From @osos on March 14, 2015 19:34_

Please consider to integrate the www.fotorama.io slideshow. It is a jquery gallery.

It has support for touch devices as well.
##

_Copied from original issue: owncloud/gallery#94_

**Comment by oparoz** [comment]
The one thing I see it has missing is zooming capabilities, but it does have fullscreen mode.
Ideally, this should be supported in parallel and users could select the one they prefer to use, because I only see this working well for pictures, not necessarily for other image types.

**Comment by oparoz** [comment]
_From @setnes on May 16, 2015 16:59_

PhotoSwipe might be an option for this as well.  It's simple, clean, and looks great on mobile devices.

http://photoswipe.com/

**Comment by oparoz** [comment]
Thanks, it could be a viable replacement as it supports most features we already use and is available via bower.

**Comment by oparoz** [comment]
_From @setnes on May 16, 2015 18:15_

Maybe I should have opened a new issue.  Osos was specifically requesting Fotorama.  I'd like to be less specific and just request that when viewing photos on a mobile device the user is allowed to swipe through the photos instead of clicking on the prior and next icons.  I'm not sure if it matters if it is something like PhotoSwipe, Photorama, or enhancements to the existing code.  Another issue that might be corrected with PhotoSwipe or Photorama is that rotating a phone or resizing any browser window doesn't result in the image being corrected to match the new size or orientation.

**Comment by oparoz** [comment]
Yes, please open 2 new issues. One for the swipe and one for the rotation :)

**Comment by oparoz** [comment]
_From @setnes on May 16, 2015 18:36_

Done.  :)

Swipe Issue:  https://github.com/owncloud/galleryplus/issues/141
Resize/Rotate Issue:  https://github.com/owncloud/galleryplus/issues/142

**Comment by oparoz** [comment]
Thanks!

**Comment by tnyeanderson** [comment]
We should be using [nextcloud/viewer](https://github.com/nextcloud/viewer) for this.

**Comment by skjnldsv** [comment]
Superseded by https://github.com/nextcloud/gallery/issues/540
